docs/documentation/cn/administrator-guide/alter-table/alter-table-temp-partition.md

```diff
@@ -0,0 +1,117 @@
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# 临时分区
+
+在 0.12 版本中，Doris 支持了临时分区功能。临时分区是对 Doris 中**已存在的数据表分区**的一个复制。这个复制操作仅复制分区的元数据，如分区值范围，分桶数，副本数等等，而**不复制数据**。
```

Comment:
I don't think the temporary partition should be a metadata copy of an existing partition. How do I accomplish the following things? How to split a particularly large partition into multiple small partitions? Or how to merge multiple partitions into one large partition?
So my point is that temporary partitions should not be bound to existing partitions, they should exist independently. Including the temporary partition should have a separate partition name, and this name has nothing to do with the existing visible partition.

Reply:
**Tmp partition is mainly for overwrite load, and it should alive very short time.**
For the partition merge and split, we could disable partition merge and split if there is tmp partition, like Alter operation.

Reply:
@kangkaisen 
I think you may have misunderstood what I meant. What I want to emphasize is not how to deal with temporary partitions during split and merge. I want to emphasize that we can also complete the functions of partition split and merge through temporary partitions. So what I mean is that we should not assume that it is related to an existing partition when doing temporary partitioning.
Currently we can just generate from other partitions, but this premise should not be made.

Reply:
> we can also complete the functions of partition split and merge through temporary partitions.

What you mean is temp partitions need to support split and merge ?

Reply:
> What you mean is temp partitions need to support split and merge ?

No.
I think we can complete the partition split and merge functions through temporary partitions. For example, we create a temporary partition and import the partition data that needs to be merged into this new temporary partition. When the data is imported, replace the previous partitions with this partition. In the same way, partition split can also be achieved through this mechanism.

Reply:
Oh. My misunderstood...
I see and I agree with you ！

Reply:
Let me think.